src/task_processor/migrations/sql/0013_get_recurringtasks_to_process.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE FUNCTION get_recurringtasks_to_process()
+RETURNS SETOF task_processor_recurringtask AS $$
+DECLARE
+    row_to_return task_processor_recurringtask;
+BEGIN
+    -- Select the tasks that needs to be processed
+    FOR row_to_return IN
+        SELECT *
+        FROM task_processor_recurringtask
+        -- Add one minute to the timeout as a grace period for overhead
+        WHERE is_locked = FALSE OR (locked_at IS NOT NULL AND locked_at < NOW() - timeout + INTERVAL '1 minute')
+        ORDER BY last_picked_at
```

Comment:
Just this line was changed here

Reply:
As per comment above (which incorrectly references the django ORM), if we make the column nullable, this should be 

```suggestion
        ORDER BY last_picked_at NULLS FIRST
```